stages/apps/sharedsvc/kube-prometheus-stack.tf

```diff
@@ -6,16 +6,16 @@ module "monitoring_namespace" {
   source    = "../../../modules/common/namespace"
   namespace = var.monitoring_namespace
   annotations = {
-    name                                         = var.monitoring_namespace
-    cluster                                      = var.cluster_name
+    name    = var.monitoring_namespace
+    cluster = var.cluster_name
```

Comment:
Is cluster_name needed for prometheus? Just wondering, as we'd managed to avoid adding it thus far